tests/invert_test_gtest.hpp

```diff
@@ -205,6 +206,7 @@ INSTANTIATE_TEST_SUITE_P(MultiShiftEvenOdd, InvertTest,
                                  no_schwarz),
                          gettestname);
 
+#ifdef QUDA_MMA_AVAILABLE
```

Comment:
How about we skip this these tests rather than comment them out if MMA isn't available? e.g., `GTEST_SKIP`

Reply:
a3e633450853ed8c31bcc392632b69adb61cdf1e